src/main/java/ca/bc/hlth/mohorganizations/OrganizationsController.java

```diff
@@ -61,6 +61,7 @@ public Collection<Organization> getOrganizations() {
 
     @PostMapping(value = "/organizations", consumes = MediaType.APPLICATION_JSON_VALUE)
     public ResponseEntity<Object> addOrganization(@RequestBody Organization organization) {
+        logger.info("Add org endpoint was triggered " + organization.getOrganizationName());
```

Comment:
Was this intentional?

Reply:
yes, just to have some difference between images